src/components/AssetHeader/AssetHeader.tsx

```diff
@@ -3,8 +3,11 @@ import type { AccountId, AssetId } from '@shapeshiftoss/caip'
 import isEqual from 'lodash/isEqual'
 import { useMemo } from 'react'
 import { AssetIcon } from 'components/AssetIcon'
+import { Seo } from 'components/Layout/Seo'
```

Comment:
OCD me would say capitalize this but obviously fine

Reply:
yah it bugs me too, i'll change it real quick.